azure/services/managedclusters/spec.go

```diff
@@ -414,6 +432,18 @@ func (s *ManagedClusterSpec) Parameters(ctx context.Context, existing interface{
 		}
 	}
 
+	if s.HTTPProxyConfig != nil {
+		managedCluster.HTTPProxyConfig = &containerservice.ManagedClusterHTTPProxyConfig{
+			HTTPProxy:  s.HTTPProxyConfig.HTTPProxy,
+			HTTPSProxy: s.HTTPProxyConfig.HTTPSProxy,
+			TrustedCa:  s.HTTPProxyConfig.TrustedCA,
+		}
+
+		if s.HTTPProxyConfig.NoProxy != nil {
+			managedCluster.HTTPProxyConfig.NoProxy = &s.HTTPProxyConfig.NoProxy
+		}
```

Comment:
Could you add a couple unit tests to `spec_test.go` to cover these cases?

Reply:
sure.